src/Core/Context/CurrentContext.cs

```diff
@@ -224,7 +224,7 @@ private List<CurrentContextOrganization> GetOrganizations(Dictionary<string, IEn
                 }));
         }
 
-        if (claimsDict.ContainsKey(Claims.OrganizationManager))
+        if (!UseFlexibleCollections && claimsDict.ContainsKey(Claims.OrganizationManager))
```

Comment:
If we skip this and don't add any claim in its place, it's going to look like the user isn't a member of the organization at all.

If FC is enabled, you could add a `CurrentContextOrganization` with `Type = OrganizationUserType.User`, however there are still other sources (e.g. db sprocs) that will be returning the "real" value. So should we try to overwrite it like this, or just update the logic that checks it?

Reply:
> If we skip this and don't add any claim in its place, it's going to look like the user isn't a member of the organization at all.

You're absolutely right! I've reverted this change.

> Where is this consumed?

Only on the method `CurrentContext.OrganizationManager` so I've modified it to throw an exception if the feature flag is enabled.